models/issue.go

```diff
@@ -71,6 +71,8 @@ type Issue struct {
 	// IsLocked limits commenting abilities to users on an issue
 	// with write access
 	IsLocked bool `xorm:"NOT NULL DEFAULT false"`
+	// IsPrivate limits who can see the issue.
+	IsPrivate bool `xorm:"NOT NULL DEFAULT false"`
```

Comment:
Is a migration needed for this field? despite the provided xorm tag?

Reply:
A migration is still needed because they run before the db sync. So if a future migration were to touch this field it would fail if it doesn't exist.